content/en/docs/concepts/windows/user-guide.md

```diff
@@ -3,38 +3,44 @@ reviewers:
 - jayunit100
 - jsturtevant
 - marosset
-title: Guide for scheduling Windows containers in Kubernetes
-content_type: concept
+title: Guide for Running Windows Containers in Kubernetes
+content_type: tutorial
 weight: 75
 ---
 
 <!-- overview -->
 
-Windows applications constitute a large portion of the services and applications that run in many organizations.
-This guide walks you through the steps to configure and deploy Windows containers in Kubernetes.
+This page provides a walkthrough for some steps you can follow to run
+Windows containers using Kubernetes.
+The page also highlights some Windows specific functionality within Kubernetes.
+
+It is important to note that creating and deploying services and workloads on Kubernetes
+behaves in much the same way for Linux and Windows containers.
+The [kubectl commands](/docs/reference/kubectl/) to interface with the cluster are identical.
+The examples in this page are provided to jumpstart your experience with Windows containers.
+
 
 <!-- body -->
 
 ## Objectives
 
-* Configure an example deployment to run Windows containers on the Windows node
-* Highlight Windows specific functionality in Kubernetes
+Configure an example deployment to run Windows containers on a Windows node.
+
+## {{% heading "prerequisites" %}}
 
-## Before you begin
+You should already have access to a Kubernetes cluster that includes a
+worker node running Windows Server.
 
-* Create a Kubernetes cluster that includes a control plane and a worker node running Windows Server
-* It is important to note that creating and deploying services and workloads on Kubernetes
-  behaves in much the same way for Linux and Windows containers.
-  [Kubectl commands](/docs/reference/kubectl/) to interface with the cluster are identical.
-  The example in the section below is provided to jumpstart your experience with Windows containers.
+The Windows node must be labelled as `kubernetes.io/os=windows`.
 
-## Getting Started: Deploying a Windows container
+## Getting Started: Deploying a Windows application
 
 The example YAML file below deploys a simple webserver application running inside a Windows container.
 
-Create a service spec named `win-webserver.yaml` with the contents below:
+Create a manifest named `win-webserver.yaml` with the contents below:
```

Comment:
should we add a note saying that the tag for the container image must match the Windows Server version for the nodes

Reply:
I think that's a separate issue best covered outside this PR.